src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.netcoreapp.cs

```diff
@@ -2,18 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+using System.Runtime.InteropServices;
 using System.Security.AccessControl;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.IO
 {
     public static partial class FileSystemAclExtensions
     {
+        #region Public methods
```

Comment:
Nit: This file is way too small. Region annotations are not useful for small files.